packages/lab/src/progress/LinearProgress/LinearProgress.css

```diff
@@ -6,26 +6,10 @@
   font-size: var(--linearProgress-fontSize);
 }
 
-/* TODO: Odyssey now has only 2 sizes */
-.saltLinearProgress-large {
-  --linearProgress-after-top: 3px;
-  --linearProgress-value-paddingLeft: 12px;
-  --linearProgress-height: 7px;
-  --linearProgress-fontSize: 16px;
-}
-
-.saltLinearProgress-medium {
-  --linearProgress-after-top: 2px;
-  --linearProgress-value-paddingLeft: 8px;
-  --linearProgress-height: 5px;
-  --linearProgress-fontSize: 14px;
-}
-
-.saltLinearProgress-small {
-  --linearProgress-after-top: 1px;
-  --linearProgress-value-paddingLeft: 4px;
-  --linearProgress-height: 3px;
-  --linearProgress-fontSize: 11px;
+.saltLinearProgress {
+  --linearProgress-after-top: calc((var(--linearProgress-height) - var(--salt-size-basis-unit) * 0.5) / 2);
+  --linearProgress-value-paddingLeft: var(--salt-size-unit);
+  --linearProgress-height: var(--salt-size-detail);
```

Comment:
since this is only one value now, can use the size token directly above and delete this variable